bot/common/threads/points.py

```diff
@@ -65,8 +65,8 @@ def get_contribution_rows(contributions):
             [
                 fields.get("Activity"),
                 fields.get("status"),
-                fields.get("Date of Submission"),
-                fields.get("Date of Engagement"),
+                fields.get("DateofSubmission").split("T")[0],
```

Comment:
maybe parsing this with datetime will be safer and bubble up a helpful error

Reply:
fixed